src/main/java/org/realworld/utils/ModelValidator.java

```diff
@@ -0,0 +1,20 @@
+package org.realworld.utils;
+
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.Validation;
+import jakarta.validation.ValidatorFactory;
+import org.hibernate.validator.messageinterpolation.ParameterMessageInterpolator;
+
+import java.util.Set;
+
+public class ModelValidator {
```

Comment:
if this class is in package utils, better to name it with suffix "Utils".  
Also, methods usually are static in utils classes

Reply:
done